code/modules/mob/living/carbon/human/human.dm

```diff
@@ -431,6 +431,9 @@
 	else if(istype(AM,/obj/structure/bed/chair/vehicle/wheelchair/motorized/syndicate))
 		var/obj/structure/bed/chair/vehicle/wheelchair/motorized/syndicate/WC = AM
 		WC.crush(src,species.blood_color)
+	else if(istype(AM, /obj/item/projectile/meteor)) //OOP damnit
```

Comment:
How else am I supposed to do it ? This is literally how the Crossed() proc works